packages/config/package.json

```diff
@@ -16,6 +16,7 @@
     "@prisma/get-platform": "workspace:*",
     "@swc/core": "1.10.11",
     "@swc/jest": "0.2.37",
+    "effect": "3.12.10",
```

Comment:
This seems like a pretty big additional dependency with a lot of implications on how we write our TS code moving forward. Would love to have a more in depth discussion in the team before we introduce such a large additional dependency. Wdyt?

Reply:
(Solved in a call)